File: code-samples/python-basics/relational_operators.py
def main() -> None:
    x = 1
    print(x == 1) # Prints True
    
    print(x < 4) # Prints True
    
    print(x > 4) # Prints False

    x = 3
    y = 7
    print(x + y <= 10) # Prints True

    print(x + y >= 11) # Prints False

    print(x != 4) # Prints True

    # Some relational operators, such as ==, work on more than
    # just numeric expressions:

    my_string = 'Hello'
    print(my_string == 'Hello') # Prints True

    print(my_string != 'Hello') # Prints False

    # Of course, you can store the value of a relational
    # operation in a boolean variable:

    my_string_is_equal_to_goodbye = my_string == 'Goodbye'
    print(my_string_is_equal_to_goodbye) # Prints False

if __name__ == '__main__':
    main()

File: code-samples/cs274/c-basics/relationaloperators.c
#include <stdio.h>

int main() {
	printf("%d\n", 5 < 10); // Prints 1 for true
	printf("%d\n", 5 >= 10); // Prints 0 for false

	printf("%d\n", 5 == 5); // Prints 1 for true
	printf("%d\n", 5 != 5); // Prints 0 for false

	printf("%d\n", 5 == 6); // Prints 0 for false
	printf("%d\n", 5 != 6); // Prints 1 for true

	// Since relational operators just produce 0 or 1,
	// you can store their values in int variables
	int x;
	scanf("%d", &x);
	int x_is_less_than_10 = x < 10;

	// If the user entered a value less than 10, then
	// x_is_less_than_10 will be 1. Else, it'll be 0.
}
